wagtail/admin/views/reports.py

```diff
@@ -186,17 +186,6 @@ class ReportView(SpreadsheetExportMixin, TemplateResponseMixin, BaseListView):
     template_name = "wagtailadmin/reports/base_report.html"
     title = ""
     paginate_by = 10
-    export_heading_overrides = {
-        "latest_revision_created_at": _("Updated"),
-        "status_string": _("Status"),
-        "content_type.model_class._meta.verbose_name.title": _("Type"),
-    }
-    list_export = [
-        "title",
-        "latest_revision_created_at",
-        "status_string",
-        "content_type.model_class._meta.verbose_name.title",
-    ]
```

Comment:
This is another breaking change - the 'adding reports' example in the docs expects these to be defined on ReportView. Perhaps we need an intermediate `PageReportView` that both the docs example and LockedPagesView inherit from?